### ISSUE TITLE
Minimap occupies full viewport height even if it's shorter than the viewport height

### ISSUE THREAD

**Issue by mnquintana** [atom-minimap/minimap#337]
(This is a feature suggestion, not a bug)

![screenshot 2015-06-04 22 45 46](https://cloud.githubusercontent.com/assets/823545/7998546/815370d6-0b0b-11e5-92c5-c0fe82e429fe.png)

It would be awesome if there were an option (or a default) for a "floating mode" where the minimap "floats" above the editor instead of blocking text at a certain width (not sure how coherent this is – I should probably get some sleep :laughing:)

**Comment by abe33** [comment]
Hi @mnquintana, Yes it was requested several time now, so I guess it's probably time that I integrate it. I once proposed a CSS fix for that in that #322 that looked like this:

```css
atom-text-editor::shadow atom-text-editor-minimap {
  position: absolute;
  right: 0;
}
```

**Comment by abe33** [comment]
Ok, I didn't fully understood your proposal at first. I think pure floating (as a floating div do in a regular html page) will be impossible to achieve, because it would require to have different wrapping length for each line based on the height of the minimap. But an absolute positioning should achieve pretty much the same effect. Let me know how you feel about that.
